sensu/gomutator.go

```diff
@@ -8,18 +8,18 @@ import (
 	"os"
 )
 
-type GoMutator struct {
-	GoPlugin
+type goMutator struct {
+	basePlugin
 	out                io.Writer
 	validationFunction func(event *types.Event) error
 	executeFunction    func(event *types.Event) (*types.Event, error)
 }
 
 func NewGoMutator(config *PluginConfig, options []*PluginConfigOption,
 	validationFunction func(event *types.Event) error,
-	executeFunction func(event *types.Event) (*types.Event, error)) *GoMutator {
-	goMutator := &GoMutator{
-		GoPlugin: GoPlugin{
+	executeFunction func(event *types.Event) (*types.Event, error)) GoPlugin {
```

Comment:
Similar to the `goHandler`, this should return a `goMutator`.

Reply:
Same as above.

Reply:
Removed code assertion since `NewGoMutator` now returns a `GoMutator`.